docs/ldml/tr35-info.md

```diff
@@ -1116,87 +1116,110 @@ The examples in #4 are due to the following ordering of the `unitQuantity` eleme
+Mixed unit identifiers should be from largest to smallest (eg foot-and-inch instead of inch-and-foot). 
+If it turns out that some locales present certain mixed units in a different order, additional structure will be needed in CLDR.
+
+If a maximum/minimum number of decimals or signficant digits is desired in formatting, 
+those values should should be adapted to the mixed radix used, starting with the largest unit, 
+and discarding smaller units where they would be overly accurate. 
+For example, if the goal is 1 part in 100, then "6 miles 1 foot" is overly precise, 
+while "6 foot 1 inch" has approximately the right accuracy.
 
 ## Testing
 
-The [unitsTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testData/units/unitsTest.txt) file supplies a list of all the CLDR units with conversions, for testing implementations. Instructions for use are supplied in the header of the file.
+The files in the directory [cldr/common/testData/units/](https://github.com/unicode-org/cldr/tree/main/common/testData/units) are provided for testing implementations. 
+1. The [unitsTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testData/units/unitsTest.txt) file supplies a list of all the CLDR units with conversions
+2. The [unitPreferencesTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testData/units/unitPreferencesTest.txt) file supplied tests for user preferences
+3. The [unitLocalePreferencesTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testData/units/unitLocalePreferencesTest.txt) file provides examples for testing the interactions between locale identifiers and unit preferences. 
+
+Instructions for use are supplied in the header of the file.
 
 ## <a name="Unit_Preferences" href="#Unit_Preferences">Unit Preferences</a>
 
 Different locales have different preferences for which unit or combination of units is used for a particular usage, such as measuring a person’s height. This is more fine-grained than merely a preference for metric versus US or UK measurement systems. For example, one locale may use meters alone, while another may use centimeters alone or a combination of meters and centimeters; a third may use inches alone, or (informally) a combination of feet and inches.
 
 ### <a name="Unit_Preferences_Overrides" href="#Unit_Preferences_Overrides">Unit Preferences Overrides</a>
 
-The determination of preferred units depends on the locale identifer: the keys mu, ms, rg, their values, the base locale (language, script, region) and the user preferences data.
+The determination of preferred units uses the user preference data together with **input unit**, the **input usage**, and the **input locale identifer**.
+Within the locale identifier, the subtags that can affect the result are:
+  * the value of the keys mu, ms, and rg
```

Comment:
sd?

Reply:
We don't currently have anything in the spec about using sd in order to get a hint about the region. Was that implemented in ICU or ICU4X?

Reply:
I thought we started using `sd` in the `-u-fw` resolution algorithm.

In ICU4X, I wrote infrastructure to handle data keyed on `-u-sd`, but nothing in CLDR currently utilizes that code path. It seems like it might want to do so in the future; for example, it's easy to imagine Scottland using different units for something like road mileage than Wales.

Reply:
I think that might be the case. The spec provides for subdivision values in the unit preferences data, but as yet there is no such data. I'll file a ticket for adding that in v46 (we'll have to work out some of the details).

https://unicode-org.atlassian.net/browse/CLDR-17474